library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -249,8 +267,10 @@ internal class TransactionPayloadFragment :
                     result.add(TransactionPayloadItem.BodyLineItem(SpannableStringBuilder.valueOf(it)))
                 }
             } else {
-                bodyString.lines().forEach {
-                    result.add(TransactionPayloadItem.BodyLineItem(SpannableStringBuilder.valueOf(it)))
+                if (!bodyString.isBlank()) {
```

Comment:
```suggestion
                if (bodyString.isNotBlank()) {
```
Also why is this needed? `.lines()` should be empty if the bodyString is blank.

Reply:
It is not empty. It has 1 item with just an empty string. I added this check so I could decide if the payload is empty.